application-tracking.md

```diff
@@ -0,0 +1,49 @@
+# Application Tracking
+
+## Principles
+
+### Why?
+
```

Comment:
I thought the "why" could do with some high level context.

Tracking is really about communication - it is a protocol for the developers, via the data team, to tell the business how their changes work (and in the middle here is being able to read data in an automated and meaningful way). 

Tracking is not just about "web tracking" but covers multiple mechanisms of performance monitoring.  

Can I suggest the following:

We build applications to support the business and drive revenue. 

To associate the performance of applications and the behaviour of customers with commercial performance and revenue, we instrument our applications with _Event Tracking_.

_Events_ form sequences of activity (clicked X, viewed Y), which can be correlatated with outcomes (made a booking, added an upgrade).
Event sequences can be separated into different streams, or segments (by specific user, by product, by version of the application).  This sequences form the foundation of business monitoring.

**Tracking is a fundamental communication mechanism between application development and commercial performance.**

Events can come from many sources, not just explicit web tracking but also application logfiles, operational databases or external systems, such as Responsys.